### ISSUE TITLE
Tag sidebar not Showing on Mobile device

### ISSUE THREAD

**Issue by Frankstar** [nebulade/meemo#79]
![meemo_tagsidebar](https://user-images.githubusercontent.com/176627/31534719-675e2de4-aff8-11e7-945d-20c90af589e6.png)

Im Not sure how to handle this.
Maybe its the best way not showing it. (like its right now)
Or only Show Pinned Tags ? (-> #51)

or create on mobile a seperate box/menu where the Tag sidebar / pinneds are showed?

**Comment by nebulade** [comment]
Yes I guess a sidebar would be best here. Overall the tag listing UI is kind an afterthought at the moment. If there are any ideas here, let me know. I will experiment a bit as well.
